install-introduce-self-git.Rmd

```diff
@@ -30,7 +30,7 @@ What user name should you give to Git? This does not have to be your GitHub user
 
 What email should you give to Git? This __must__ be the email associated with your GitHub account.
 
-These commands return nothing. You can check that Git understood what you typed by looking at the output of `git config --global --list`.
+The first two commands from 7.0 (above) using `git config --global use.[something]` return nothing in the terminal. You can check that Git understood what you typed by looking at the output of `git config --global --list`.
```

Comment:
I'm afraid I can't merge something that hard-wires the chapter/section numbering. You can either have another crack at the wording (something about "the commands that set the configuration don't ...") or I'll do so next time I work on Happy Git.